package.json

```diff
@@ -164,14 +166,17 @@
     "serve-static": "^1.10.0",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
-    "tmp": "0.0.31"
+    "tmp": "0.0.31",
+    "ts-node": "^2.0.0",
+    "typescript": "^2.1.5"
   },
   "scripts": {
-    "build-local-continuous": "babel src --watch -d lib --ignore '**/*_spec.js'",
-    "build-local": "babel src -d lib --ignore '**/*_spec.js'",
+    "build-local-continuous": "babel src --watch -d lib --ignore '**/*_spec.js' --extensions .js",
+    "build-local": "babel src -d lib --ignore '**/*_spec.js' --extensions .js && cp src/typings/cucumber.d.ts lib/index.d.ts",
```

Comment:
We are also using an `&&` in `yarn test` and that works on [appveyor](https://ci.appveyor.com/project/charlierudolph/cucumber-js/build/1.0.720/job/58l26lttys34s700)